jpo-ode-plugins/src/main/java/us/dot/its/jpo/ode/plugin/RoadSideUnit.java

```diff
@@ -45,8 +59,22 @@ public RSU(String rsuTarget, String rsuUsername, String rsuPassword, int rsuRetr
             this.rsuRetries = rsuRetries;
             this.rsuTimeout = rsuTimeout;
 
-            // default to 4.1 SNMP protocol
-            this.snmpProtocol = SnmpProtocol.FOURDOT1;
+            String defaultSnmpProtocol = System.getenv("DEFAULT_SNMP_PROTOCOL");

```

Comment:
Let's abstract this snmp set to a separate method to avoid the duplicated code here. Maybe just "setSnmpProtocol" or something.

Reply:
Pulled this out to a 'setDefaultSnmpProtocol' method as 'setSnmpProtocol' is already present